app/src/main/java/com/example/alpaki/di/main/ExampleActivityLevelModule.kt

```diff
@@ -14,4 +15,8 @@ object ExampleActivityLevelModule {
     @Provides
```

Comment:
Actually, both UseCases can be constructor-injected :)

Reply:
fixed, removed both providers